src/pages/Order/Order.js

```diff
@@ -0,0 +1,189 @@
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import Icons from '../../components/Icons/Icons';
+import Modal from './Modal/Modal';
+import './Order.scss';
+
+const Order = () => {
+  const [orderData, setOrderData] = useState([]);
+  const [modalOpen, setModalOpen] = useState(false);
+  const navigate = useNavigate();
+
+  const [requirements, setRequirements] = useState({
+    delivery: false,
+    terms: false,
+  });
+
+  const { delivery, terms } = requirements;
+
+  useEffect(() => {
+    fetch('/data/orderData.json', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+      },
+    })
+      .then(response => response.json())
+      .then(result => {
+        setOrderData(result);
+      });
+  }, []);
+
+  const formNotComplete = delivery === false || terms === false;
+
+  const { message } = orderData;
+
+  return (
+    <div className="order">
+      {modalOpen && (
+        <Modal
+          modalOpen={() => setModalOpen(!modalOpen)}
+          setModalOpen={setModalOpen}
+          data={message}
+        />
+      )}
+
+      <div className="section-top">
+        <div className="logo" />
+      </div>
+      <div className="section-main">
+        <div className="section-text">
+          <div className="section-header">
+            <div className="step-header">배송과 픽업 방법</div>
+          </div>
+          <div className="delivery-address">배송지:</div>
+          <div className="address-text">{message?.userInfo[0].addresses}</div>
+          <div onClick={() => navigate('/cart')} className="fix-address">
+            수정
+          </div>
+          <div className="delivery-type-header">배송 방법</div>
+          <div className={`delivery-option-box ${delivery && `active`}`}>
+            <div className="title-wrap">
+              <div className="delivery-option-title">무료 배송</div>
+              <div className="delivery-option-title">₩ 0</div>
+            </div>
+            <div className="delivery-time">1-7일</div>
+            <div className="delivery-time">배송일 - 가장 빠른 날짜</div>
+            <div className="delivery-time">
+              {new Date().toLocaleDateString('en-US', {
+                year: 'numeric',
+                month: '2-digit',
+                day: '2-digit',
+              })}
+            </div>
+            <div className="no-contact-box">
+              <div className="delivery-option-title">비대면 배송</div>
+              <div className="checkbox-container">
+                <input
+                  onClick={() =>
+                    setRequirements({ ...requirements, delivery: !delivery })
+                  }
+                  type="checkbox"
+                  className="checkbox"
+                />
+                <div className="checkbox-label">제품을 현관 앞에 놔두기</div>
+              </div>
+            </div>
+          </div>
+          <div className={!delivery && 'inactive'}>
```

Comment:
```suggestion
          <div className={!delivery ? 'inactive' : ''}>
```
질문 주신 부분에서 에러가 발생하는 부분이 이 부분으로 보여집니다!
기존 코드에서는 앞에 있는 값이 true가 아닐경우 실제 코드에는
```
<div className=false >
```
이러한 형태처럼 부여되기에 에러가 발생하게 됩니다!
없을 때 아무것도 부여하지 않을 때는 삼항연산자를 사용하는 것으로 적용해주세요!